drivers/driver.go

```diff
@@ -100,6 +100,13 @@ type ProtoDriver interface {
 	Cleanup() error
 	// AdditionalImageStores returns additional image stores supported by the driver
 	AdditionalImageStores() []string
+
+	// MountTemp mounts a source directory from the host using
+	// graphdriver and returns the mountpoint
+	MountTemp(sourcedir, source, mountLabel string) (string, error)
```

Comment:
Which parameter is the directory?  What's the other parameter used for?

Reply:
Fixed by renaming sourcedir to contentDir.